man/systemd.journal-fields.xml

```diff
@@ -56,10 +56,9 @@
     their syntax but with fields that can include binary data.
     Primarily, fields are formatted UTF-8 text strings, and binary
     formatting is used only where formatting as UTF-8 text strings
-    makes little sense. New fields may freely be defined by
-    applications, but a few fields have special meaning. All fields
-    with special meanings are optional. In some cases, fields may
-    appear more than once per entry.</para>
+    makes little sense. New fields may freely be defined (multiple times)
+    by applications. A few fields have special meaning. All fields
+    with special meanings are optional and usually just set once.</para>
```

Comment:
"Usually" is usually not good in reference texts. It would be good to be more precise here. What about "systemd-journald will attach special fields (starting with a single underscore). Those fields occur a single time."?